sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1207,6 +1207,8 @@ class Analyzer(
         val relation = table match {
           case u: UnresolvedRelation if !u.isStreaming =>
             resolveRelation(u).getOrElse(u)
+          case r: V2TableReference =>
```

Comment:
Can this ever be `DataSourceV2Relation` itself? I am asking primarily as there is `resolveAsV2Relation` that handles both below.

Reply:
We already have `ResolveInsertInto` to handle it.